components/core/SlateMediaObjectPreview.js

```diff
@@ -18,7 +18,8 @@ const STYLES_IMAGE_CONTAINER = css`
 const STYLES_IMAGE = css`
   background-color: ${Constants.system.foreground};
   display: block;
-  max-width: 100%;
+  width: 100%;
+  height: 100%;
```

Comment:
height: 100%; does not do anything here, especially because its a block level element.